utils/checkfiles.py

```diff
@@ -90,28 +90,33 @@ def get_auto_conf_images(full_tpl=False):
 
     # walk through the auto-config dir
     for yaml_file in os.listdir(auto_confd_path):
-        check_name = yaml_file.split('.')[0]
-        try:
-            # load the config file
-            auto_conf = check_yaml(urljoin(auto_confd_path, yaml_file))
-        except Exception as e:
-            log.error("Enable to load the auto-config, yaml file.\n%s" % str(e))
-            auto_conf = {}
-        # extract the supported image list
-        images = auto_conf.get('docker_images', [])
-        for image in images:
-            if full_tpl:
-                init_tpl = auto_conf.get('init_config') or {}
-                instance_tpl = auto_conf.get('instances', [])
-                if image not in auto_conf_images:
-                    auto_conf_images[image] = [[check_name], [init_tpl], [instance_tpl]]
-                else:
-                    for idx, item in enumerate([check_name, init_tpl, instance_tpl]):
-                        auto_conf_images[image][idx].append(item)
-            else:
-                if image in auto_conf_images:
-                    auto_conf_images[image].append(check_name)
+        # Ignore files that do not end in .yaml
+        extension = yaml_file.split('.')[-1]
+        if extension != 'yaml':
```

Comment:
`.yml` is parsed successfully, however, [we look for the file](https://github.com/DataDog/dd-agent/pull/3150/files#diff-305e19f343add5fc856b48d568ee8d97R63) by adding `.yaml` on to `check_name`.

 This assumes all config files end in `.yaml`. The same is true for files in `/conf.d`, just tested with `docker_daemon.yml` and it was not picked up.